heimdall2/values.yaml

```
@@ -154,32 +195,101 @@ heimdall:
                 port:
                   number: 3000
 
-    # annotations:
-    #   traefik.ingress.kubernetes.io/router.entrypoints: websecure
-    #   traefik.ingress.kubernetes.io/router.tls: "true"
-    #   traefik.ingress.kubernetes.io/service.sticky.cookie: "true"
-
-    # className: ingress class name
-
-    # tls:
-    #   - name: nginx
-    #     secretName: ingress-secret
-    #     hosts:
-    #       - heimdall.example.com
+#     annotations:
+#       traefik.ingress.kubernetes.io/router.entrypoints: websecure
+#       traefik.ingress.kubernetes.io/router.tls: "true"
+#       traefik.ingress.kubernetes.io/service.sticky.cookie: "true"
+
+#     className: ingress class name
+
+#     tls:
+#       - name: nginx
+#         secretName: ingress-secret
+#         hosts:
+#           - heimdall.example.com
+  
+  gateway:
+#    enabled: false
```

enable this attribute set to false so that the gateway thing isn't just sitting there without any contents

```suggestion
   enabled: false
```